secret_number.js

```diff
@@ -1,4 +1,7 @@
 'use strict';
-module.exports = function() {
-  
+module.exports = function(){
+    var secretNum = Math.floor(Math.random() * 1000000);
```

Comment:
You should multiple `Math.random()` with 1000001 or else you'll never reach 1000000 since `Math.random()` will generate a random number between 0 and .9999999999. 

Also, use const here.